olc

```diff
@@ -32,9 +41,7 @@ class Orchestrator:
                 line = input()
                 if line is None:
                     break
-                result = self.execute(line)
-                if result:
-                    print(result)
+                self.execute(line)
```

Comment:
With this change I don't think the output will be printed to the terminal when running the repl, but I think that it should be printed. Can you let me know the intent of this change